v2/pkg/nodenet/pod.go

```diff
@@ -213,17 +215,30 @@ func (pn *podNetwork) Setup(nsPath, podName, podNS string, conf *PodNetConf, hoo
 			return fmt.Errorf("netlink: failed to get veth link for container: %w", err)
 		}
 
+		lo, err := netlink.LinkByName("lo")
+		if err != nil {
+			return fmt.Errorf("netlink: failed to get lo: %w", err)
+		}
+
 		idx := 0
 		if conf.IPv4 != nil {
 			ipnet := netlink.NewIPNet(conf.IPv4)
-			err := netlink.AddrAdd(cLink, &netlink.Addr{
+			err := netlink.AddrAdd(lo, &netlink.Addr{
 				IPNet: ipnet,
 				Scope: unix.RT_SCOPE_UNIVERSE,
 			})
 			if err != nil {
 				netlink.LinkDel(cLink)
 				return fmt.Errorf("netlink: failed to add an address: %w", err)
 			}
+			err = netlink.AddrAdd(cLink, &netlink.Addr{
+				IPNet: &net.IPNet{IP: containerIPv4, Mask: net.CIDRMask(30, 32)},
```

Comment:
`net.CIDRMask(30, 32)`

This should be defined as a constant, I mean, a variable used as a constant.